build.gradle

```
@@ -259,6 +266,180 @@ testClusters.integTest {
     }
 }
 
+String bwcVersion = "1.13.0.0";
+String baseName = "adBwcCluster"
+String bwcFilePath = "src/test/resources/org/opensearch/ad/bwc/"
+
+testClusters {
+    "${baseName}" {
+        testDistribution = "ARCHIVE"
+        versions = ["7.10.2","1.0.0"]
+        numberOfNodes = 3
+        plugin(provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "job-scheduler/" + bwcVersion).getSingleFile()
+                    }
+                }
+            }
+        }))
+        plugin(provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "anomaly-detection/" + bwcVersion).getSingleFile()
+                    }
+                }
+            }
+        }))
+        setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+        setting 'http.content_type.required', 'true'
+    }
+}
+
+List<Provider<RegularFile>> plugins = [
+        provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "job-scheduler/" + project.version).getSingleFile()
+                    }
+                }
+            }
+        }),
+        provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "anomaly-detection/" + project.version).getSingleFile()
+                    }
+                }
+            }
+        })
+    ]
+
+//Creates a test cluster with 3 nodes of the old version.
```

super minor nit: space after comment escape string.
```suggestion
// Creates a test cluster with 3 nodes of the old version.
```

---

> super minor nit: space after comment escape string.

Updated for all the comments.